crates/driver/src/infra/solver/mod.rs

```diff
@@ -129,6 +130,8 @@ pub struct Config {
     pub solver_native_token: ManageNativeToken,
     /// Which `tx.origin` is required to make quote verification pass.
     pub quote_tx_origin: Option<eth::Address>,
+    /// List of surplus capturing JIT-order owners
+    pub surplus_capturing_jit_order_owners: HashSet<H160>,
```

Comment:
Types in the domain should use the domain types meaning this would be `HashSet<eth::Address>`. But since another comment suggests passing the cow amm addresses in the `/solve` request this would go away anyway.